includes/scheduler/class-post.php

```diff
@@ -51,28 +51,26 @@ function ( $post_id ) {
 	public static function schedule_post_activity( $new_status, $old_status, $post ) {
 		$post = get_post( $post );
 
-		if ( ! $post || is_post_disabled( $post ) ) {
+		if ( ! $post ) {
```

Comment:
`is_post_disabled()` needs to be moved down so `ap_extrafield` and `ap_extrafield_blog` posts are not considered disabled and can be processed.

Reply:
It was pragmatic to add the `ap_extrafield` to the hook, but technically it should live in the actor scheduler... should we keep it here for now?

Reply:
I think I'd prefer to move it to the Actor scheduler